apps/penxle.com/src/lib/components/forms/Radio.svelte

```diff
@@ -0,0 +1,27 @@
+<script lang="ts">
+  import clsx from 'clsx';
+  import { getFormContext } from '$lib/form';
+
+  let _class: string | undefined = undefined;
+  export { _class as class };
+  export let name: string | undefined = undefined;
+  export let checked: boolean | undefined = undefined;
```

Comment:
```suggestion
  type $$Props = HTMLInputAttributes;

  let _class: $$Props['class'] = undefined;
  export { _class as class };
  export let name: $$Props['name'] = undefined;
```

P3: 컴포넌트 props 타입 정의가 필요해요!